Update multi CSI driver related docs (#399)
harvester/docs: docs/advanced/storageclass.md

:::note

> The provisoner of the VM image still use Longhorn. Before v1.2.0, Harvester only supports the use of Longhorn to store virtual machine data, and does not support the use of external storage to store VM data.

Reviewer: ```suggestion
The provisioner of the VM image still uses Longhorn. Before v1.2.0, Harvester only supports using Longhorn to store virtual machine data and does not support using external storage to store VM data.
```

:::

Administrators can enable this feature through the following steps:
1. Set the [`os.persistent_state_paths`](../install/harvester-configuration.md#ospersistent_state_paths) and [`os.after_install_chroot_commands`](../install/harvester-configuration.md#osafter_install_chroot_commands) settings before creating the Harvester cluster for the specific CSI driver needs.
2. Install the CSI driver to the Harvester cluster manually after creating the Harvester cluster.
3. Edit the [`csi-driver-config`](../advanced/settings.md#csi-driver-config) setting.
4. Create a new StorageClass that uses the CSI driver.

> After that, users just need to select the desired StorageClass when creating an empty volume or adding a new empty volume to a virtual machine.

Reviewer: ```suggestion
After performing these steps, you must select the desired StorageClass when creating an empty volume or adding a new empty volume to a virtual machine.
```
